config/settings.yml

```diff
@@ -0,0 +1,8 @@
+hca:
+  endpoint: https://test-foo.vets.gov
```

Comment:
Went with a fake url since the VA is touchy about real endpoints being in a public repo. This can be overridden for dev by putting something like this into `config/settings.local.yml`:

```yaml
hca:
  endpoint: https://some-real-endpoint.va.gov
```

There was also some weird name conflict with the EVSS stuff, hence `https://test-foo.vets.gov` instead of just `https://test.vets.gov`